mongodb-atlas-sql-interface.md

```diff
@@ -40,22 +40,25 @@ To connect using the Atlas SQL interface :
 
 3. Select **Database** from the categories on the left, select **MongoDB Atlas SQL**, and then select **Connect**.
 
-2. In the MongoDB Atlas SQL window that appears provide:
+4. If this is the first time you're connecting to MongoDB Atlas SQL connector, a third-party notice will be displayed. 
+   Select Don't warn me again with this connector if you don't want this message to be displayed again, 
+   and then select **Continue**. 
+
+5. In the MongoDB Atlas SQL window that appears provide:
    - The **MongoDB URI** you retrieved previously. Make sure that it does not contain your username and password or it 
```

Comment:
```suggestion
   - _Required_. The **MongoDB URI**. _Required_. 
   
      Use the MongoDB URI you obtained in step 1.  Make sure that it does not contain your username and password. URIs containing username and/or passwords will be rejected.
```

Reply:
I think you did not mean to have _required_ twice. I couldn't find how to edit your suggestion, but I made the change to mark it required 👍